app/views/home/about.html.erb

```diff
@@ -0,0 +1,74 @@
+<div class="container about-page">
+  <div class="row about-intro">
+
+      <div class="col-md-12">
+          <h1 class="intro-title">We made Giving Profiles to help you help others</h1>
```

Comment:
If you're following BEM this would be something like `about-intro__title`. Your block is the about intro, and your element is the title w/in the block.